src/lib.rs

```diff
@@ -230,6 +230,8 @@ pub use near_indexer_primitives;
 pub use aws_credential_types::Credentials;
 pub use types::{LakeConfig, LakeConfigBuilder};
 
+use s3_fetchers::LakeClient;
```

Comment:
I would suggest calling this `LakeS3Client` and giving the corresponding trait `S3Client` name instead.

Reply:
Done